apps/spruce/src/gql/fragments/projectSettings/viewsAndFilters.graphql

```diff
@@ -5,7 +5,6 @@ fragment ProjectViewsAndFiltersSettings on Project {
     exactMatch
     expression
   }
-  projectHealthView
```

Comment:
Should we create a ticket to remove this setting? It was designed specifically for the Project Health page... and then no one ever used it. 😭😭

Reply:
:( yes! I'll do it along side the other mainline commits deprecation work